unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -350,6 +351,76 @@ def _trigger_upload_content(
 
         return Content(**created_content)
 
+    def send_download_content_from_chat_request(
```

Comment:
The function name is a bit complex in my opinion or even misleading. How about `request_content` or `get_content`?